communication/marketing-team/social-guidelines.md

```diff
@@ -31,15 +32,33 @@ Kubernetes Organization Members with access to account are:
 - Significant historical events as related to the project (Kubernetes birthday, previous events, etc.)
 - Steering Committee Announcements
 
+### Example Social Media
+
+- Weekly on Sunday: Link with community calendar and "This week in upstream community group meetings..."
+- As they come in:
+  - Emails from k-dev
+  - #Shoutouts (as requested; please include social handles)
```

Comment:
This should be pulled from the shoutout channels in slack either daily or every two days, etc some kind of cadence. Might want to include that in here to be descriptive for future folks.